src/test/regress/expected/citus_local_tables.out

```diff
@@ -361,6 +361,23 @@ SELECT create_citus_local_table('local_table_3');
 
 (1 row)
 
+-- test creating citus local table with an index from non-default schema
+CREATE SCHEMA test_index_schema;
```

Comment:
lets' make this schema a more interesting one so that we can test if we're escaping special characters (like "test_\'index_schema")